assets.md

```diff
@@ -19,6 +19,7 @@ This file is intended to list all the assets controlled by OpenTelemetry.
     - Ask any of the following people if you need to manage the feed:
     - Owner: Amye Scavarda Perrin (CNCF rep)
     - Managers: Sergey Kanzhelev, Austin Parker
+    - Maintainer: Eddy Nakamura
```

Comment:
I think you can just add yourself as a manager. I think I mistakenly called it Maintainer in offline comms.

Reply:
done.